sorc/link_workflow.sh

```diff
@@ -122,7 +122,11 @@ for dir in aer \
     rm -rf "${dir}"
   fi
   fix_ver="${dir}_ver"
-  ${LINK_OR_COPY} "${FIX_DIR}/${dir}/${!fix_ver}" "${dir}"
+  if [[ "${dir}" == "aer" ]]; then
+    ${LINK_OR_COPY} "${FIX_DIR}/${dir}" "${dir}"
+  else
+    ${LINK_OR_COPY} "${FIX_DIR}/${dir}/${!fix_ver}" "${dir}"
+  fi
```

Comment:
The `aer` fix subtype is already being linked. See line 104 and remove this change.

Reply:
@AnningCheng-NOAA `${!fix_ver}` is still needed and should be left as is. To use the new `aer` version please update `aer_ver` in `versions/fix.ver`. Thanks!

Reply:
done